doc/how_to_guides/isaac_panda/isaac_panda_tutorial.rst

```diff
@@ -0,0 +1,169 @@
+
+For us to expand our Panda robot to Isaac Sim we first have to introduce `isaac_ros2_control <https://github.com/PickNikRobotics/isaac_ros2_control>`_.
+This Hardware Interface is a ``System`` that subscribes and publishes on a configured topics.
+For this tutorial the topic ``/isaac_joint_states`` will be the robots current state and ``/isaac_joint_commands`` will be used to actuate it.
+The `moveit_resources_panda_moveit_config <https://github.com/ros-planning/moveit_resources/blob/humble/panda_moveit_config/config/panda.ros2_control.xacro#L7>`_
+we are using in this tutorial does not support connecting to hardware, so our ``ros2_control.xacro`` is now
+updated to load the ``IsaacSystem`` plugin when the flag ``ros2_control_hardware_type`` is set to ``isaac``.
+
+.. code-block:: XML
+
+    <xacro:if value="${ros2_control_hardware_type == 'mock_components'}">
+        <plugin>mock_components/GenericSystem</plugin>
+    </xacro:if>
+    <xacro:if value="${ros2_control_hardware_type == 'isaac'}">
+        <plugin>isaac_ros2_control/IsaacSystem</plugin>
+        <param name="joint_commands_topic">/isaac_joint_commands</param>
+        <param name="joint_states_topic">/isaac_joint_states</param>
+    </xacro:if>
+
+In this tutorial we have included an Isaac python script that loads a Panda robot
+and builds an `OnmiGraph <https://docs.omniverse.nvidia.com/prod_extensions/prod_extensions/ext_omnigraph.html>`_
+to publish and subscribe to the ros topics used to control the robot.
+To learn about configuring your Isaac robot to communicate with ROS2 please see the
+`Joint Control tutorial <https://docs.omniverse.nvidia.com/app_isaacsim/app_isaacsim/tutorial_ros2_manipulation.html>`_
+on Omniverse.
+
+Computer Setup
+--------------
+
+1. Install `Isaac Sim <https://docs.omniverse.nvidia.com/app_isaacsim/app_isaacsim/install_workstation.html>`_.
+
+2. For RViz to run inside Docker we have to tell the host to allow it to display applications.
+
+.. note:: this step is only required once every host machine login
+
+.. code-block:: bash
+
+  xhost +local:docker
+
+1. Perform a shallow clone of the MoveIt2 Tutorials repo.
+
+.. code-block:: bash
+
+  git clone https://github.com/ros-planning/moveit2_tutorials.git -b humble --depth 1
+
+1. Cd into the path you cloned the tutorials and then switch to the following directory.
+
+.. code-block:: bash
+
+  cd moveit2_tutorials/doc/how_to_guides/isaac_panda
+
+5. Build the Docker image.
+
+.. code-block:: bash
+
+  docker compose build
+
+6. Start a container based on the new image.
+
+.. code-block:: bash
+
+  docker compose up
+
+You should see the name of the container started and a prompt to build the colcon workspace if needed.
+
+.. code-block:: bash
+
+  [+] Running 1/1
+  ⠿ Container isaac_panda-base-1
+  Attaching to isaac_panda-base-1
+  isaac_panda-base-1  | Sourced ROS 2 Humble
+  isaac_panda-base-1  | Please build the isaac_moveit_tutorial workspace with
+  isaac_panda-base-1  | colcon build
```

Comment:
If the Dockerfile builds the workspace for the user, and we're not mounting anything, is this step even needed?

EDIT: Confirmed this did not happen on my end, so you can probably remove this step.